src/js/fetchBooks.js

```diff
@@ -0,0 +1,13 @@
+import Notiflix from 'notiflix';
+
+export async function fetchBooks() {
```

Comment:
this function returns request only for top-books, per documentation we have different requests that we need 
![Screenshot 2024-03-01 at 21 26 44](https://github.com/spaceTony9/temp_name/assets/128310401/ccd7b657-f33e-40a1-ae46-bb7e9689ccfd)